cmd/manager/main.go

```diff
@@ -111,6 +110,11 @@ func main() {
 		setupLog.Error(err, "unable to set up client config")
 		os.Exit(1)
 	}
+	clientSet, err := kubernetes.NewForConfig(cfg)
```

Comment:
Can we put inline comment on for what this is for just like other objects we are creating ?

Reply:
done